docs/API/metadata.md

```diff
@@ -23,20 +23,25 @@ based on the owner's retention policy and the server administrator's retention
 policy along with some other internal management context. The expected deletion
 date is accessible under the `server` metadata key namespace.
 
-Clients can also set arbitrary metadata through the "dashboard" and "user"
+Clients can also set arbitrary metadata through the "global" and "user"
 metadata namespaces:
-* The "dashboard" namespace can only be modified by the owner of the dataset,
+* The "global" namespace can only be modified by the owner of the dataset,
 and is visible to anyone with read access to the dataset.
 * The "user" namespace is private to each authenticated user, and even if you
 don't own a dataset you can set your own private "user" metadata to help you
 categorize that dataset and to find it again.
 
 Metadata namespaces are hierarchical, and are exposed as nested JSON objects.
-You can address an entire namespace, e.g., `dashboard` or `dataset` and
+You can address an entire namespace, e.g., `global` or `dataset` and
 retrieve the entire JSON object, or you can address nested objects or values
-using a dotted metadata key path like `dashboard.contact.email` or
+using a dotted metadata key path like `global.contact.email` or
 `dataset.metalog.pbench.script`.
 
+By convention, a Pbench Server client should create a sub-namespace to minimize
+the risk of key collisions within the `global` and `user` namespaces. The
+Pbench Dashboard client, for example, uses `global.dashboard.seen` and
+`user.dashboard.favorite`.
```

Comment:
I wonder if we should use a funkier spelling, like `pbdashboard` -- if all goes well, ours might not be the only "dashboard" client.